chunky/src/java/se/llbit/chunky/ui/SceneChooserController.java

```diff
@@ -173,7 +178,11 @@ public void setStage(Stage stage) {
   private void populateSceneTable(File sceneDir) {
     List<SceneListItem> scenes = new ArrayList<>();
     List<File> fileList = SceneHelper.getAvailableSceneFiles(sceneDir);
-    Collections.sort(fileList);
+    // TODO: Sort by recently modified instead of file name.
```

Comment:
Doesn't this PR do just that? Also, the user should be able to change the sorting order by clicking on the table column head… it's weird that we force the sorting.

Reply:
oop, yeah. I added that in at the beginning before I wrote the rest. I'll have to remove that.